extensions/amp-story/1.0/amp-story.js

```diff
@@ -1031,10 +1034,15 @@ export class AmpStory extends AMP.BaseElement {
   /**
    * Switches to a particular page.
    * @param {string} targetPageId
+   * @param {string} direction
```

Comment:
If the direction will only ever be a limited set of options, consider using a Closure Compiler `@enum` type instead of a raw string.

https://github.com/ampproject/amphtml/blob/5e57e9f72807ecdb832b9bc00d044dd6a8df41f0/extensions/amp-story/1.0/page-advancement.js#L42-L46

Reply:
Done, in the amp-story-page level.